src/Cake.Core/CakeEnvironment.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
```

Comment:
sorry about that, i'll have done some reorganizing. will try and update PR later tonight